connectors/services/base.py

```diff
@@ -17,5 +20,5 @@ async def run(self):
         raise NotImplementedError()
 
     def shutdown(self, sig):
-        logger.info(f"Caught {sig.name}. Graceful shutdown.")
+        logger.info(f"Caught {sig.name}. Shutdown {self.__class__.__name__}.")
```

Comment:
let's remove `Caught {sig.name}` this should be in the `_service_shutdown`

Reply:
OK, will do.

Reply:
I have done it in this PR: https://github.com/elastic/connectors-python/pull/337